aws-globalaccelerator-accelerator/src/test/kotlin/software/amazon/globalaccelerator/accelerator/ListHanderTest.kt

```diff
@@ -0,0 +1,151 @@
+        val request = ResourceHandlerRequest.builder<ResourceModel>().build()
+        val response = ListHandler().handleRequest(proxy, request, null, logger)
+        assertNotNull(response)
+        assertEquals(OperationStatus.SUCCESS, response.status)
+        assertNull(response.callbackContext)
+        assertNull(response.resourceModel)
+        assertNull(response.message)
+        assertNotNull(response.resourceModels)
+        assertNull(response.nextToken)
+        assertEquals(2, response.resourceModels.size)
+        assertEquals(acceleratorArn1, response.resourceModels[0].acceleratorArn)
+        assertEquals(model1.ipAddressType, response.resourceModels[0].ipAddressType)
+        assertEquals(model1.name, response.resourceModels[0].name)
+        assertEquals(model1.dnsName, response.resourceModels[0].dnsName)
+        assertTrue(response.resourceModels[0].enabled)
+        assertEquals(model1.ipAddresses, response.resourceModels[0].ipAddresses)
+        assertEquals(acceleratorArn2, response.resourceModels[1].acceleratorArn)
+        assertEquals(model2.ipAddressType, response.resourceModels[1].ipAddressType)
+        assertEquals(model2.name, response.resourceModels[1].name)
+        assertEquals(model2.dnsName, response.resourceModels[1].dnsName)
+        assertTrue(response.resourceModels[1].enabled)
+        assertEquals(model2.ipAddresses, response.resourceModels[1].ipAddresses)
+    }
+
+    @Test
+    fun handleRequest_noAccelerators() {
+        val listAcceleratorsResult = ListAcceleratorsResult().withAccelerators(emptyList())
+        every { proxy.injectCredentialsAndInvoke(ofType(), ofType<ProxyListAccelerators>()) } returns listAcceleratorsResult
+
+        val request = ResourceHandlerRequest.builder<ResourceModel>().build()
+        val response = ListHandler().handleRequest(proxy, request, null, logger)
+        assertNotNull(response)
+        assertEquals(OperationStatus.SUCCESS, response.status)
+        assertNotNull(response.resourceModels)
+        assertEquals(0, response.resourceModels.size)
+        assertNull(response.nextToken)
+    }
+
+    @Test
+    fun handleRequest_returnsMappedAcceleratorsWithToken() {
+        val expectedNextToken = "This_token_is_expected"
+        val accelerators = mutableListOf(
+                Accelerator()
+                        .withAcceleratorArn(acceleratorArn1)
+                        .withStatus(AcceleratorStatus.IN_PROGRESS.toString())
+                        .withEnabled(true)
+                        .withIpAddressType(ipFamily)
+                        .withName(name1)
+                        .withDnsName(dnsName1)
+                        .withIpSets(ipSet1)
+        )
+        val listAcceleratorsResult = ListAcceleratorsResult()
+                .withAccelerators(accelerators)
+                .withNextToken(expectedNextToken)
+        every { proxy.injectCredentialsAndInvoke(ofType(), ofType<ProxyListAccelerators>()) } returns listAcceleratorsResult
```

Comment:
Does this actually cover the case confirming the request downstream has the token?  Looks like it just makes sure we actually make the functionc all, but not that the downstream args are correct.